### PR TITLE
Added Horizontal Mentor card

### LINKED ISSUE #105: Create a separate component for mentor card (horizontal view)
**Is your feature request related to a problem? Please describe.**
In the admin's mentee profile view, we are currently displaying the mentor details like this:
image

Currently, it is built inside the MentorProfile component, rather than as a separate component.

**Describe the solution you'd like**
Make it a separate component. Consider other pages where this component can be reused and adjust the design accordingly.

**Describe alternatives you've considered**
N/A

**Additional context**
Bug introduced with #103

### PR DESCRIPTION
**Pull request by dilankavishka**
## Purpose

The purpose of this PR is to fix #105 

## Goals
This PR introduces a separate component for the mentor card in horizontal view, as opposed to keeping it inside the MentorProfile component. The goal is to improve reusability across the project and maintain cleaner code.

## Approach
* Created a new, reusable MentorCardHorizontal component.

## Screenshot
Screenshot (47)

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [-] I have commented my code, particularly in hard-to-understand areas
- [-] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by anjula-sack** [comment]
can you add a screenshot? Also the pipeline is failing @dilankavishka

**Comment by dilankavishka** [comment]
@anjula-sack, I did the changes

File: src/components/MenteeApplication/MenteeApplication.component.tsx [hunk 1]
```diff
@@ -137,31 +138,9 @@ const MenteeApplication: React.FC = () => {
...
-                    </h3>
-                    <p className="line-clamp-2 text-sm text-gray-700">
-                      {mentee?.mentor.application.position},{' '}
-                      {mentee?.mentor.application.institution}
-                    </p>
-                  </div>
-                </div>
+                <MentorCardHorizontal
+                  mentor={mentee?.mentor}>
+                </MentorCardHorizontal>
```
Comment (anjula-sack) [inline src/components/MenteeApplication/MenteeApplication.component.tsx:143]:
```suggestion
                <MentorCard mentor={mentee?.mentor} />
```

File: src/components/MentorCardHorizontal/MentorCardHorizontal.component.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        />
+      </div>
+    <div>
+      <h3 className="font-medium sm:text-lg">
+        <a
+          href="#"
+          className="hover:underline"
+          target="_blank"
+          rel="noreferrer"
+        >
```
Comment (anjula-sack) [inline src/components/MentorCardHorizontal/MentorCardHorizontal.component.tsx:27]: Link to the mentor page

File: src/components/MentorCardHorizontal/MentorCardHorizontal.component.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { type Mentor } from '../../types.ts';
+import ProfilePic from '../ProfilePic/index.tsx';
+
+interface MentorCardProps {
+  mentor: Mentor|undefined;
```
Comment (anjula-sack) [inline src/components/MentorCardHorizontal/MentorCardHorizontal.component.tsx:6]:
```suggestion
  mentor?: Mentor;
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Pull request closed without merging**